docs/xxidletuningcompactonidle.md

```diff
@@ -26,23 +26,23 @@
 
 **(Linux&reg; only)**
 
-This option controls garbage collection processing with compaction when the status of the OpenJ9 VM is set to idle.
+This option controls garbage collection processing with compaction when the OpenJ9 VM is running in a container and the state of the VM is set to idle.
```

Comment:
To me this update gives an impression that the option is applicable only when OpenJ9 is running in container, which is not correct. I feel this does not need any change.